.eslintignore

```diff
@@ -1,3 +1,3 @@
 */node_modules/*
-bundle-size/*.example.js
-test-status/*.example.js
+bundle-size/*.example.js  
+test-status/*.example.js
```

Comment:
Pleasy change your editor's config to add a newline at the end of files >____<

Reply:
It used to autoformat and somehow I broke that =__= I'll get it figured out